component/exporter.go

```diff
@@ -68,20 +68,29 @@ type ExporterFactory interface {
 	// tests of any implementation of the Factory interface.
 	CreateDefaultConfig() config.Exporter
 
-	// CreateTracesExporter creates a trace exporter based on this config.
+	// CreateTracesExporter creates a TracesExporter based on this config.
 	// If the exporter type does not support tracing or if the config is not valid,
 	// an error will be returned instead.
 	CreateTracesExporter(ctx context.Context, set ExporterCreateSettings, cfg config.Exporter) (TracesExporter, error)
 
-	// CreateMetricsExporter creates a metrics exporter based on this config.
+	// TracesExporterStability gets the stability level of the TracesExporter.
+	TracesExporterStability() StabilityLevel
```

Comment:
Is the advantage to breaking out stability level by signal and component type that we no longer need to pass in a DataType to get the stability?

Reply:
also, this pattern works for Extensions which don't have a data type.

Reply:
> which don't have a data type.

makes sense.

Reply:
Thanks.